docs/existing-aws-infrastructure.md

```diff
@@ -7,45 +7,27 @@ Normally, Cluster API will create infrastructure on AWS when standing up a new w
 In order to have Cluster API consume existing AWS infrastructure, you will need to have already created the following resources:
 
 * A VPC
-* One or more private subnets (subnets that do not automatically assign a public IP address to EC2 instances)
+* One or more private subnets (subnets that do not have a route to an Internet gateway)
 * A public subnet in the same Availability Zone (AZ) for each private subnet (this is required for NAT gateways to function properly)
 * A NAT gateway for each private subnet, along with associated Elastic IP addresses
 * An Internet gateway for all public subnets
 * Route table associations that provide connectivity to the Internet through a NAT gateway (for private subnets) or the Internet gateway (for public subnets)
 
-You will need the following information, which can be obtained either via the AWS Management Console or using the AWS CLI:
+Note that a public subnet (and associated Internet gateway) are required for the control plane of the workload cluster.
```

Comment:
I think I've mislead you – we insist that there _be_ public subnets, but it looks like we [know not to use them for the control plane load balancer ](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/9913ac8dbb96ed9bcffa6eba75b886cd4ef8c20b/pkg/cloud/services/elb/loadbalancer.go#L288) if the scheme is set to `internal`. Maybe:

```suggestion
Note that a public subnet (and associated Internet gateway) are required even if the control plane of the workload cluster is set to use an `internal` load balancer.
```

?

Reply:
I've adopted your change (included in 1667e3ab). Is it worthwhile including information on the mechanism by which an internal load balancer is configured, or is that information best suited to be elsewhere?